src/apps/concepts/components/ViewConceptsHeader.tsx

```diff
@@ -84,6 +97,27 @@ const ViewConceptsHeader: React.FC<Props> = ({
       } else setPreferredSources(defaultSources);
     }, [showSources, sources, dictionaries]);
 
+    const fetchMoreData = () =>{
+      setTimeout(()=>{
+        const previousSources =preferredSources?.filter(Boolean).concat(Array.from({length:10}));
+        setPreferredSources(previousSources)
+     
+      },1000);
```

Comment:
This looks like some sort of simulated code? I think before we get all of these elements looped in we need to start smaller, with just either the search or the toggle. It might be easiest, oddly to just start with a search box and show a default page of 10 results or w/e.